test_collections/matter/scripts/build-local-sdk-image.sh

```
@@ -0,0 +1,174 @@
+# Builds the SDK image (chip-cert-bins) locally for the host architecture,
+# tagged exactly as the backend expects. Needed on non-arm64 hosts (e.g. WSL),
+# where the published image is not available.
+#
+# Usage: build-local-sdk-image.sh [dockerfile_ref]
+#   dockerfile_ref  Optional git ref of the connectedhomeip repo to fetch the
+#                   Dockerfile from. Defaults to the pinned SDK_DOCKER_TAG.
+#                   Pass "master" if the Dockerfile at the pinned tag fails to
+#                   build (e.g. due to fixes that landed after the tag).
+
+set -e
+MATTER_PROGRAM_DIR=$(realpath $(dirname "$0")/..)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)